mse_home/command/spawn.py

```diff
@@ -63,14 +64,16 @@ def add_subparser(subparsers):
         "--size",
         type=int,
         required=True,
-        help="The enclave size to spawn (in MB, must be a power of 2)",
+        help="The enclave size to spawn",
+        choices=[4096, 8192, 16384, 32768, 65536],
     )
 
     parser.add_argument(
         "--signer-key",
         type=Path,
-        required=True,
+        required=False,
         help="The enclave signer key",
+        default="/opt/cosmian-internal/cosmian-signer-key.pem",
```

Comment:
See https://gramine.readthedocs.io/en/latest/manpages/gramine-sgx-gen-private-key.html#description.
```suggestion
        default=f"{os.getenv('HOME', '/root')}/.config/gramine/enclave-key.pem",
```